ballerina/build.gradle

```diff
@@ -82,20 +84,26 @@ dependencies {
 
 task updateTomlFiles {
     doLast {
-        def newConfig = ballerinaConfigFile.text.replace("@project.version@", project.version)
+        def newConfig = ballerinaTomlFilePlaceHolder.text.replace("@project.version@", project.version)
         newConfig = newConfig.replace("@toml.version@", tomlVersion)
         newConfig = newConfig.replace("@kafka.version@", kafkaVersion)
         ballerinaConfigFile.text = newConfig
 
-        def newPluginConfig = ballerinaCompilerPluginFile.text.replace("@project.version@", project.version)
+        def newPluginConfig = compilerPluginTomlFilePlaceHolder.text.replace("@project.version@", project.version)
         ballerinaCompilerPluginFile.text = newPluginConfig
     }
 }
 
-task revertTomlFiles {
+task commitTomlFiles {
     doLast {
-        ballerinaConfigFile.text = originalConfig
-        ballerinaCompilerPluginFile.text = originalCompilerPlugin
+        project.exec {
+            ignoreExitValue true
+            if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+                commandLine 'cmd', '/c', "git commit Ballerina.toml CompilerPlugin.toml -m '[Automated] Commit the native jar versions'"
+            } else {
+                commandLine 'sh', '-c', "git commit Ballerina.toml CompilerPlugin.toml -m '[Automated] Commit the native jar versions'"
```

Comment:
```suggestion
                commandLine 'sh', '-c', "git commit Ballerina.toml CompilerPlugin.toml -m '[Automated] Update the native jar versions'"
```

Reply:
Changed in https://github.com/ballerina-platform/module-ballerinax-kafka/pull/280/commits/2d0a36ee6ac39bd2f6eec146a23726b1eea39abe